faker/providers/person/az_AZ/__init__.py

```diff
@@ -0,0 +1,728 @@
+from .. import Provider as PersonProvider
+
+
+def translit(text):
+    translit_dict = {
+        'Ə': 'E', 'I': 'I', 'Ü': 'U', 'Ş': 'SH', 'Ç': 'CH', 'Ğ': 'G', 'Ö': 'O',
+        'ə': 'e', 'ı': 'i', 'ü': 'u', 'ş': 's', 'ç': 'c', 'ğ': 'g', 'ö': 'o',
+    }
+    for letter in text:
+        if letter.isalpha():
+            text = text.replace(letter, translit_dict[letter])
+    return text
+
+
+class Provider(PersonProvider):
+    formats_female = (
+        '{{first_name_female}} {{last_name_female}}',
+        '{{first_name_female}} {{first_name_male}}',
+        '{{first_name_female}} {{last_name_unisex}}',
+    )
+
+    formats_male = (
+        '{{first_name_male}} {{last_name_male}}',
+        '{{first_name_male}} {{first_name_male}}',
+        '{{first_name_male}} {{last_name_unisex}}',
+    )
+
+    formats = formats_male + formats_female
+
+    first_names_female = ('Adilə',
```

Comment:
do you have a source for these names?

Reply:
I gathered them from different sources, and also made manual changes to them. I will add resources.